abis/mlibc/mount.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _ABIBITS_MOUNT_H
+#define _ABIBITS_MOUNT_H
+
+#define MNT_FORCE 1
+
+#endif // _ABIBITS_MOUNT_H
```

Comment:
Why does mlibc get its own ABI even though it matches the Linux ABI anyway?

Reply:
For consistency's sake I suppose.

Reply:
That doesn't make sense, sysdeps can just link to `abis/linux` instead of `abis/mlibc`.